File: 6-Tarea-Arreglos-matrices/A.Reverso.cpp
#include <iostream>
using namespace std;

int main() {
  int N;
  cin >> N;

  int arreglo[N];

  for (int i = 0; i < N; i++) 
    cin >> arreglo[i];
  
  for (int i = N - 1; i >= 0; i--)
    cout << arreglo[i] << " ";

  return 0;
}

File: 6-Tarea-Arreglos-matrices/A.Reverso.cs
using System.Collections.Generic;
using System.Linq;
using System;

class Program
{
  static void Main(string[] args)
  {
    int N = int.Parse(Console.ReadLine());
    int[] arreglo = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();

    for (int i = N - 1; i >= 0; i--)
      Console.Write(string.Format("{0} ", arreglo[i]));
  }
}